MediaBrowser.Model/IO/StreamDefaults.cs

```diff
@@ -13,6 +13,6 @@ public static class StreamDefaults
         /// <summary>
         /// The default file stream buffer size
         /// </summary>
-        public const int DefaultFileStreamBufferSize = 81920;
+        public const int DefaultFileStreamBufferSize = 4096;
```

Comment:
Why?

Reply:
Because that is the default value for thz `FileStream` buffer size. And this value wasn't used before this PR afaik so no existing uses change.
Ref: https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/IO/FileStream.cs#L17